scripts/qesap/pylint.rc

```diff
@@ -214,8 +214,8 @@ min-public-methods=2
 [EXCEPTIONS]
 
 # Exceptions that will emit a warning when caught.
-overgeneral-exceptions=BaseException,
-                       Exception
+#overgeneral-exceptions=BaseException,
```

Comment:
I commented it out as I noticed a warning during the tox execution (when performed it locally)

```
pylint: commands[0]> pylint --rcfile=pylint.rc qesap.py lib/
pylint: Command line or configuration file:1: UserWarning: Specifying exception names in the overgeneral-exceptions option without module name is deprecated and support for it will be removed in pylint 3.0. U
se fully qualified name (maybe 'builtins.BaseException' ?) instead.
pylint: Command line or configuration file:1: UserWarning: Specifying exception names in the overgeneral-exceptions option without module name is deprecated and support for it will be removed in pylint 3.0. U
se fully qualified name (maybe 'builtins.Exception' ?) instead.
```